app/javascript/chat/chat.jsx

```diff
@@ -1778,24 +1776,17 @@ export default class Chat extends Component {
           if (e.keyCode === 13) this.triggerActiveContent(e);
         }}
         tabIndex="0"
-        href={path}
+        href="#/"
```

Comment:
Should this have changed?

Reply:
It shouldn't have, I actually didn't edit this but when I did `git merge master`, this merge conflict appeared.